RELEASE/scripts/autoscend.ash

```diff
@@ -4449,77 +4449,97 @@ boolean LX_bitchinMeatcar()
 		set_property("lastDesertUnlock", my_ascensions());
 		return false;
 	}
-
-	int meatRequired = 100;
-	if(item_amount($item[Meat Stack]) > 0)
-	{
-		meatRequired = 0;
-	}
-	foreach it in $items[Spring, Sprocket, Cog, Empty Meat Tank, Tires, Sweet Rims]
+	
+	//craft meatcar if you have all the components. note that you also need 70 meat for 7 meat pastes if not in knoll sign
+	if(!knoll_available() && item_amount($item[Tires]) > 0 && item_amount($item[spring]) > 0 && item_amount($item[sprocket]) > 0 && item_amount($item[cog]) > 0 && item_amount($item[empty meat tank]) > 0)
```

Comment:
It seems like all we're doing here is:
* If we don't have Knoll access, tally up the meat-pasting cost
* If we do have Knoll access, tally up the cost of buying the components. Otherwise, fail if we don't have the components.
* Fail if we don't have enough meat.
* Make the car.

Can we split this logic into something like `int meatForPasting` and `int meatForComponents` and do checks like `if (my_meat() >= meatForPasting + meatForComponents`? This would allow us to combine the two `make bitch` forks and (I suspect) make the code cleaner?

Also you might be able to use `creatable_amount()` here to clean this up

Reply:
handled via commit fd192a2